api/v1alpha3/machineset_webhook.go

```diff
@@ -78,3 +86,29 @@ func (m *MachineSet) validate() error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineSet").GroupKind(), m.Name, allErrs)
 }
+
+// PopulateDefaultsMachineSet fills in default field values.
+// This is also called during MachineSet sync.
```

Comment:
Is this comment accurate? I'm not seeing any other callers of `PopulateDefaultMachineSet`

Reply:
all fixed!